lib/recipients/project.js

```diff
@@ -44,6 +45,21 @@ module.exports = async ({ schema, logger, task }) => {
     logger.verbose('licence is granted, notifying applicant');
     notifications.set(applicantId, { emailTemplate: 'licence-granted', applicant, recipient: applicant });
 
+    const additionalEstablishments = await ProjectEstablishment.query().where({ projectId, status: 'active' });
+
+    if (additionalEstablishments.length) {
+      additionalEstablishments.reduce((promise, pe) => {
+        return promise
+          .then(() => getAdmins(pe.establishmentId))
+          .then(admins => {
+            logger.verbose('Licence granted, notifying additional availability admins');
+            admins.map(profile => {
+              notifications.set(profile.id, { emailTemplate: 'licence-granted', applicant, recipient: profile });
```

Comment:
Need to make sure that the establishment id is correctly set in the link in the email. 

```suggestion
              notifications.set(profile.id, { emailTemplate: 'licence-granted', applicant, recipient: profile, establishmentId:  pe.establishmentId});
```

Reply:
good point